packages/uikit-workshop/webpack.config.js

```diff
@@ -349,22 +331,19 @@ module.exports = function(apiConfig) {
       },
       plugins: [
         // clear out the buildDir on every fresh Webpack build
-        new CleanWebpackPlugin(
-          config.watch
+        new CleanWebpackPlugin({
+          verbose: false,
+          cleanOnceBeforeBuildPatterns: config.watch
             ? []
             : [
                 `${config.buildDir}/index.html`,
                 `${config.buildDir}/styleguide/css`,
                 `${config.buildDir}/styleguide/js`,
               ],
-          {
-            allowExternal: true,
```

Comment:
@sghoweri Do we really (still) need this? Introduced with https://github.com/pattern-lab/patternlab-node/blob/3d38d4399bc8553b68bf5914fc94af3351bb49d6/packages/uikit-workshop/webpack.config.js#L213 - `allowExternal` got replaced by `dangerouslyAllowCleanPatternsOutsideProject` with a newer version of the plugin, see https://github.com/johnagan/clean-webpack-plugin/issues/106

Reply:
If the build is still ok and the preview loads up correctly after building the ui-kit I would assume we do not need the parameter anymore.